RELEASES.md

```diff
@@ -1,3 +1,125 @@
+Version 1.49.0 (2020-11-19)
+============================
+
+Language
+-----------------------
+
+- [Unions now implement `Drop`, and you can now have a field in a union
+  with `ManuallyDrop<T>`.][77547]
+- [You can now cast zero sized enums (0 or 1 variants) integers.][76199]
```

Comment:
Uninhabited is a very different thing from zero-sized, and https://github.com/rust-lang/rust/pull/76199 supports casts for uninhabited enums.